### ISSUE TITLE
[FAB] Ensure ARIA roles, states, and properties are valid (CAL-AMP37) 

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#13538]
**monday.com sync:** #10821967560

### Violation: 
 Ensure ARIA roles, states, and properties are valid 

### Recommended solution
The component's `busy` and `live` attributes should be removed.

### WCAG Reference: 
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html) 

### Severity: 
 High 

### Media Type: 
 ARIA 

--- 
### Areas for Remediation: 
- Title: CAL-AMP37 Ensure ARIA roles, states, and properties are valid 
- Module: M5 Stepper, Tabs, Link, FAB 
- Finding ID: [CAL-AMP37](https://esri.hub.essentia11y.com/short-link/MhtBPMPmXp8zoL1p) 

**Issue**
There is a button (present in the "FAB" section) with unnecessary aria-live and aria-busy attributes.

**User Impact**
Assistive technologies may be unable to parse this content, so users may not be able to perceive the feature(s) or control(s) correctly, or may not be able to operate them.

**Code Reference** 
 ```
<button aria-live="polite" aria-busy="false" aria-label="" class=" button-padding--shrunk icon-end-empty " href="false" rel="false" target="false" type="button">(...)</button>

```

**Steps to Reproduce**
1. Open Chrome DevTools.
2. Locate the element identified in the code reference.
3. Notice that the element's ARIA attributes and/or nesting do not conform to the ARIA specification: https://www.w3.org/TR/wai-aria/ and/or the ARIA in HTML specification: https://www.w3.org/TR/html-aria/
 
**Recommendation**
Ensure ARIA roles, states, and properties are valid. 

In this case, the developer should remove aria-live and aria-busy attributes from the ```<button>``` element.

**Compliant Code Example**
```
<img src="oldbarn.png" alt="Old barn near Jenny Lake" aria-details="imgDetails">
...
<p id="imgDetails">
An old barn with a tin roof and weathered siding stands bathed in morning sunlight near Jenny Lake with the Wyoming Rocky Mountains in the background.
</p>
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP37.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/MhtBPMPmXp8zoL1p) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.
